config/v1alpha1/register.go

```diff
@@ -38,6 +38,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&ImagePolicyList{},
 		&ClusterImagePolicy{},
 		&ClusterImagePolicyList{},
+		&UpdateStatus{},
+		&UpdateStatusList{},
```

Comment:
[Failing CI][1].  Did you mean to drop these now that you're mostly in `update/`?

[1]: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_api/2012/pull-ci-openshift-api-master-unit/1843324895635705856#1:build-log.txt%3A40

Reply:
yeah forgot to drop these when I moved things to `update`